### PR TITLE
Home

### PR DESCRIPTION
**Pull request by harshalini**
Added basic routing and home page in notes app
Screenshot 2022-04-29 113505
preview -
https://626b7f25cb245930b6599fc5--reliable-strudel-9a7e90.netlify.app/

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,10 +1,14 @@
+@import url("https://liniui.netlify.app/component.css");
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@400;500;700&display=swap');
+@import url(./styles/home.css);
 body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```
Comment (sanket4120) [inline src/index.css]: remove this font-family

File: src/pages/home.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Navbar } from "../components/allComp"
+export const Home = () => {
+    return (
+        <div>
+            <Navbar />
```
Comment (sanket4120) [inline src/pages/home.js]: put navbar in app file before routes as it will be required in all components not only homepage

**Review by sanket4120 — APPROVED** [review]
Rest looks good

**Review by AnkurChunekar — APPROVED** [review]
- Other than the changes suggested by @sanket4120 , 
- everything else looks good to merge.

File: src/pages/home.js (hunk 2 above)
Comment (harshalini) [reply src/pages/home.js]: Ok thanks for suggestion

**Pull request merged**